building-a-preview/index.md

```diff
@@ -0,0 +1,777 @@
+  `canceled` status in red.
+
+### Service states
+
+A Service could be in any of the above states, as well as:
+
+- **Committing** - Tugboat is currently taking a snapshot of the current state
+  of the Service.
+- **Waiting** - Tugboat is performing some operation on the Service's repo, and
+  the Service is waiting for its turn.
+
+## Set a Base Preview
+
+If you want to speed up your Preview builds, and make subsequent Preview builds
+smaller files, you can set a Base Preview as a starting point for subsequent
+Previews.
+
+- [How Base Previews work](#how-base-previews-work)
+- [How to set a Base Preview](#how-to-set-a-base-preview)
+- [Using multiple Base Previews](#using-multiple-base-previews)
+- [Building a Preview from scratch after you've set a Base Preview](#building-a-preview-from-scratch-after-youve-set-a-base-preview)
+
+### How Base Previews work
+
+When you build a regular Preview, the
+[configuration file](../setting-up-services/index.md#create-a-tugboat-config-file)
+typically instructs Tugboat to pull in databases, image files, or other assets.
+This process can take a while; the larger the assets, the longer the build.
+
+After your Preview has finished building, Tugboat automatically takes a
+point-in-time snapshot of its disk image, so it has a point of reference of
+where it can do things like let you quickly [reset](#reset) a Preview back to
+its original build state. You can leverage this snapshot to create a Base
+Preview.
+
+When you mark a Preview as a Base Preview, Tugboat can use this as a starting
+point for all subsequent newly-created Previews. None of the new Previews need
+to re-download copies of databases, image files, or other assets. Base Previews
+can dramatically reduce the amount of time required to generate a working
+Preview.
+
+In addition to speeding up your Preview builds, Tugboat saves disk space by
+storing only a binary difference between the Base Preview and Previews built
+from that Base Preview. A new Preview only uses whatever space it needs that
+differs from the Base Preview. Often, this means a Base Preview might use 2-3GB
+of space, and a Preview built from it might only use 100-200MB.
+
+> #### Hint:: Base Previews and Preview Build Stages
+>
+> When you set a Base Preview, new Previews you build - including Previews that
+> are built automatically from pull requests - use the Base Preview as a
+> starting point, and build only from the `build` stage. This means that if
+> you're making changes that would be processed during `init` or `update`
+> stages, or changing a Docker image, you'll need to either
+> [rebuild the Base Preview](#using-preview-actions-on-a-base-preview), or
+> [build the Preview from scratch without the Base Preview](#building-a-preview-from-scratch-after-youve-set-a-base-preview).
+> For more info, see:
+> [the build process: explained](#the-build-process-explained).
+
+### How to set a Base Preview
```

Comment:
This information should probably be moved into the "Administer Base Previews" section